easybuild/easyconfigs/c/CNS/CNS-1.31-linux-x86_64.eb

```diff
@@ -0,0 +1,33 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Binary'
+
+name = 'CNS'
+version = '1.31'
```

Comment:
@jordiblasco I was only able to see version 1.3 after filling the request for, are you sure this version is correct?

Reply:
Yes, that's the version I downloaded after filling up the request form: http://cns-online.org/cns_request/

Reply:
Yes, that was the version I got after filling up the request form: http://cns-online.org/cns_request/